### DISCUSSION TITLE
Unable to use component level css

### DISCUSSION THREAD

**Discussion by priyank-R** [vercel/next.js#18215] [Help] [answered]
I installed next using _npx create-next-app_ and created a _components_ directory in the root. 

I added 2 files, **components/Element1.js** and **components/Element1.module.css**

Inside **components/Element1.module.css**, I added the following code: 
`.xyz{
    color: green;
    background-color: red;
}`

Inside **components/Element1.js**, I added the following: 

```
import { Fragment } from "react";
import styles from './Element1.module.css'

export default function Element1(){
    return (
        <Fragment>
        <h1 className="styles.xyz">Hello there</h1>
        </Fragment>
    )
}
```

Later, I imported Element1 inside index.js and used it. 
Here's the code for index.js: 

```
import Head from 'next/head'
import Element1 from '../components/Element1'
export default function Home() {
  return (
    <div>
      <Head>
        <title>Create Next App</title>
        <link rel="icon" href="/favicon.ico" />
      </Head>

      <Element1 />
      
    </div>
  )
}
```

I'm unable to see the styling being applied in Element1 component. I have not used any external libraries since next supports `module.css` feature by default. I don't really understand what am I missing.

_My version of next is 9.5.5._

Please help !

**Comment by jamesmosier** [comment, marked as answer]
You will need to use your `styles` like this for all CSS modules:

```js
import styles from './Element1.module.css'

...

<h1 className={styles.xyz}>Hello there</h1>
```
